docs/user/reference/metrics/timespan.md

```diff
@@ -174,11 +174,11 @@ def on_login():
 </div>
 <div data-lang="Rust" class="tab">
 
-```rust
-use fog::metrics;
+```Rust
+use glean_metrics::auth;;
```

Comment:
Hrm. Does this mean we should rewrite FOG's metrics API generation to appear in a `glean_metrics` crate instead of in the `metrics` mod of the `fog` crate? I wouldn't mind it being consistent to support e.g. standalone WR builds linking to a full Glean SDK outside of gecko build.

Reply:
I mostly did _not_ want to make a decision about the precise naming here, but instead go with what we used more already, which was `glean_metrics`.
Now neither of these is really more correct than the other, because in the end it depends on the application.
And heck, because it's a module in every other potential rust consumer other than FOG, this line wouldn't even work (it would need to be `use crate::glean_metrics::...`).

I'd be ok to revisit this in a followup, maybe just fully removing it? or adding comments about it.

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1771707